src/Program.fs

```diff
@@ -519,9 +535,10 @@ let generate (configFile: string) =
 
                 let packageReferences = [
                     yield! packageReferences
+                    yield MSBuildXElement.PackageReferenceInclude("System.Net.Http.Json", SystemNetHttpJsonVersion)
                     yield
                         match config.serializer with
-                        | SerializerType.System -> MSBuildXElement.PackageReferenceInclude("System.Text.Json", SystemTextJsonVersion)
+                        | SerializerType.System -> MSBuildXElement.PackageReferenceInclude("FSharp.SystemTextJson", FSharpSystemTextJsonVersion)
```

Comment:
I think it would be good to have both

Reply:
Well `FSharp.SystemTextJson` depends on `System.Text.Json` right? So keeping track of one dependency seems better